content/faq.md

```diff
@@ -4,10 +4,132 @@ title: Frequently Asked Questions
+ -  [Why are you running this program?](#why-are-you-running-this-program)
+ -  [Are mentoring organizations required to use the code produced by students?](#are-mentoring-organizations-required-to-use-the-code-produced-by-students)
+ -  [Can I take part in the project as a group of contributors?](#can-i-take-part-in-the-project-as-a-group-of-contributors)
+ -  [Can I submit more than one proposal?](#can-i-submit-more-than-one-proposal)
+ -  [How to write a good project proposal?](#how-to-write-a-good-project-proposal)
+ -  [How do I know that I've been accepted?](#how-do-i-know-that-ive-been-accepted)
+ -  [Can I get course credit for participating in the Summer of Haskell?](#can-i-get-course-credit-for-participating-in-the-summer-of-haskell)
+ -  [How do I become a mentor for Summer of Haskell?](#how-do-i-become-a-mentor-for-summer-of-haskell)
+ -  [Are there any other ways to support Summer of Haskell?](#are-there-any-other-ways-to-support-summer-of-haskell)
+ -  [What if I have another question?](#what-if-i-have-another-question)
 
-Some notable exceptions to Google's policies are:
+---
+
+### What is the main goal of this program?
+Our main goal is to attract people outside the ecosystem to the Haskell
+open-source code development. We hope program attendees stay with the
+open-source movement after finishing their projects to continue contributing.
+
+---
+
+### What is the timeline for the Summer of Haskell program?
+The full timeline for the 2023 edition can be found on the [homepage](/#projected-timeline).
+
+Estimates for medium-size projects: 175 hours to complete; large projects: 350
+hours to complete. The program (work schedule) can be modified individually in
+case a student's school ends late/starts early and collides with the Summer of
+Haskell timeline.
+
+---
+
+### What are the eligibility requirements for participation?
+
+ -  You must be eligible to work in the country you will reside in during the
+    program, and the United States cannot embargo the country you reside in.
+ -  You must be at least 18 years old before the program's inauguration.
+ -  Summer of Haskell cannot accept participants (contributors or mentors) from
+    Russia, Belarus, or the so-called Donetsk People's Republic ("DNR") and
+    Luhansk People's Republic ("LNR"), but there are no restrictions on
+    participants in Ukraine but outside of the DNR and LNR.
+
+---
+
+### How does the program work?
+Newcomers are mentored by experienced contributors, and learn how to work with
+open-source and improve their ability to write Haskell code. No previous open
+source experience is needed to attend the program as a would-to-be contributor,
+however you need to have some knowledge of writing Haskell code. The program is
+addressed to everyone; there's no need to be a student. All the contributors
+are paid for their work; even if the work is unfinished or the produced code
```

Comment:
I think that we actually aren't "paying for work" per se - I believe it's legally important that this be a stipend, not a payment for work. Perhaps check how Google structures theirs, and also check in with Ryan?